scaleway/resource_instance_placement_group.go

```diff
@@ -92,7 +92,7 @@ func resourceScalewayInstancePlacementGroupRead(ctx context.Context, d *schema.R
 	}, scw.WithContext(ctx))
 
 	if err != nil {
-		if is404Error(err) {
+		if is404Error(err) || is403Error(err) {
```

Comment:
This only happend when importing the default security group do we want to handle it ?